Frends.LDAP.CreateUser/Frends.LDAP.CreateUser/Definitions/Input.cs

```diff
@@ -50,6 +50,13 @@ public class Input
     /// <example>true</example>
     public bool SetPassword { get; set; }
 
+    /// <summary>
+    /// Password can be added to the unicodePwd field as a byte[] type.   
```

Comment:
Please also state the TLS/SSL requirement to use this field, if some of those are required for this field to work.